components/form/UploadImageInput.tsx

```diff
@@ -79,7 +82,7 @@ const UploadImageInput: React.FC<UploadImageProps> = ({
         type='file'
         name='image'
         onChange={(e) => imageFileUpload(e)}
-        required={isImageRequired}
+        required={!isImageUploaded}
```

Comment:
If we change `pagePathname` to `isRequired` that can then be used in this line and it keeps a pretty straightforward relation to our props without doing this weird negative evaluation.

That does mean that we will need to add that prop to the component when it's getting called in `add-item/page.tsx`:
```js
<UploadImageInput
          setImageSrc={setImageSrc}
          setError={setGeneralError}
          isRequired={true}
        />
```